airbyte-integrations/connectors/source-sentry/acceptance-test-config.yml

```diff
@@ -32,6 +31,8 @@ acceptance_tests:
   discovery:
     tests:
       - config_path: secrets/config.json
+        backward_compatibility_tests_config:
```

Comment:
why is this change backwards incompatible? in this case we should follow a breaking changes release process